test/authorization-spec.js

```diff
@@ -0,0 +1,52 @@
+var testHelper = require('../etc/test-helper.js');
+
+describe('Access control:', function() {
```

Comment:
The test cases in this spec were largely lifted (with modifications) from general-spec.js.